tests/integration/test_vid_club.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+import pandas as pd
+
+from unittest import mock
+
+from viadot.sources import VidClub
+
+
+@pytest.fixture
+def var_dictionary():
+    variables = {}
+
+    return variables
+
+
+class MockClass:
+    status_code = 200
+
+    def json():
+        df = pd.DataFrame()
+        return df
+
+
+@pytest.mark.init
+def test_create_club_class():
+    vc = VidClub()
+    assert vc
+
+
+@pytest.mark.init
+def test_default_credential_param():
+    vc = VidClub()
+    assert vc.credentials != None and type(vc.credentials) == dict
+
+
+@pytest.mark.proper
+def test_build_query_wrong_source():
+    with pytest.raises(Exception):
+        vc = VidClub()
+        query = vc.build_query(source="test")
+
+
+@pytest.mark.proper
+def test_get_response_wrong_source():
+    with pytest.raises(Exception):
```

Comment:
Please add an expected message and correct Exception. check other tests for reference.